Source/Extensions/Blazorise.Markdown/wwwroot/markdown.js

```diff
@@ -116,6 +116,13 @@ export function initialize(dotNetObjectRef, element, elementId, options) {
         nativeSpellcheck: options.nativeSpellcheck,
         parsingConfig: options.parsingConfig,
         previewClass: options.previewClass,
+        previewRender: options.usePreviewRender ? (plainText, preview) => {
+            dotNetObjectRef.invokeMethodAsync("NotifyPreviewRender", plainText).then((htmlText) => {
```

Comment:
What happens if the custom parser takes a while? For example a second or two? it's handled fine?

Reply:
Yes, it is fine. That's why I used this approach from easy-mde. It is specifically fro async operations.